packages/console/src/components/Tooltip/index.module.scss

```diff
@@ -0,0 +1,25 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  position: absolute;
+  transform: translateX(-50%) translateY(-100%);
+  border-radius: 8px;
+  background: #191c1d;
+  color: #e0e3e3;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 8%);
+  padding: _.unit(2) _.unit(3);
+  font: var(--font-body);
```

Comment:
nit: consider adding animation effects

Reply:
same issue for Guamian - I'm not sure what to add